dev-setup.sh

```
@@ -0,0 +1,124 @@
+}
+
+# Wait for replica set to initialize
+echo -e "${YELLOW}Waiting for replica set to initialize...${NC}"
+for i in {1..15}; do
+    RS_STATUS=$(docker exec todo-mongo mongosh --quiet --eval "try { rs.status().ok } catch(e) { 0 }")
+    if [ "$RS_STATUS" == "1" ]; then
+        echo -e "${GREEN}Replica set initialized successfully!${NC}"
+        break
+    fi
+    
+    if [ $i -eq 15 ]; then
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)